src/desktop/components/main_layout/templates/scripts.jade

```diff
@@ -159,11 +162,11 @@ if options.sailthru && sd.SAILTHRU_CUSTOMER_ID
     })();
 
 //- Prefetch links on hover. See: https://instant.page/
-if sd.ENABLE_INSTANT_PAGE
+if sd.ENABLE_INSTANT_PAGE && !sd.THIRD_PARTIES_DISABLED
   include instant-page
```

Comment:
This one is a little surprising. Isn't this a client-side script we choose to include for its benefits to the perceived loading times? If it's not helpful, we could choose to eliminate it, as opposed to the truly "third party" scripts that are needed for non-technical tracking/integration reasons.

Reply:
Let me do some research as I'm not entirely sure what this script is for or even if we have it enabled.